README.md

```diff
@@ -19,6 +19,10 @@ Gradebook uses the latest version of Node LTS. Please ensure you have this insta
 
 The commands you need to run are dependent on what you're trying to do.
 
- - For all tasks, you need eleventy to run - `yarn dev`
+ - For most tasks, you need gulp running - run `yarn dev` to start a live server
```

Comment:
Should this be `yarn gulp dev` ?

Reply:
I forgot to alias dev 😁 `yarn dev` should work in almost every single repository